File: examples/python/src/find_did_by_name.py
from platform_os import is_windows


if is_windows():
    from request import get
else:
    from sock import get

# PLEASE WRITE device_name
device_name = "DEVICE_NAME"

json_response = get(f"/identifiers?device_name={device_name}")

print("The response is as follows.\n")
print(json_response)

File: examples/nodejs/src/findDidByName.js
import { get } from "./lib/sock.js";

// PLEASE WRITE device_name
const device_name = "DEVICE_NAME";

(async () => {
  const response = await get(`/identifiers?device_name=${device_name}`);

  console.log("The response is as follows.\n");
  console.log(response);
})();
